examples/client/Locomotion/src/Components/RideCard/index.tsx

```diff
@@ -61,7 +67,11 @@ const CardComponent = ({ paymentMethod, businessAccountId }: CardComponentProps)
     if (isOffline) {
       return offlineIcon;
     }
+    if (isExternal) {
+      return placeholderIcon;
```

Comment:
I think, instead of doing this, we can:
```suggestion
      return null;
```

And then, in `TextRowWithIcon`, do something like:
```
if (!icon) return <View />;
```

Or:
```
if (!icon) return null;
```

Reply:
@Davidi-74 but then you won't have the placeholder icon , why like this/?

Reply:
@galinareznikk I just noticed the placeholder icon is actually empty, is this what Product wanted?

Reply:
I asked Ido about this, and he told me to use the default payment icon
`import { PaymentIcon } from 'react-native-payment-icons'`
So there’s no need for a placeholder icon.